src/main/java/livoi/swimdiary/config/TokenAuthenticationFilter.java

```diff
@@ -0,0 +1,64 @@
+package livoi.swimdiary.config;
+
+import jakarta.servlet.FilterChain;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import livoi.swimdiary.config.jwt.TokenProvider;
+import lombok.RequiredArgsConstructor;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.web.filter.OncePerRequestFilter;
+
+/**
+ * 클라이언트의 요청을 처리할 때 JWT 토큰을 확인하여 인증 정보를 설정하는 필터를 설정하기 위한 클래스입니다.
+ * 스프링 시큐리티의 OncePerRequestFilter를 상속받아 매 요청당 한 번씩 실행
+ */
+@RequiredArgsConstructor
+public class TokenAuthenticationFilter extends OncePerRequestFilter{
+
+  private final TokenProvider tokenProvider;
+  private final static String HEADER_AUTHORIZATION = "Authorization";
+  private final static String TOKEN_PREFIX = "Bearer ";
+
+  /**
+   * 클라이언트의 요청에서 Authorization 헤더를 읽어와 JWT 토큰을 확인한 후,
+   * 유효한 토큰일 경우 해당 토큰을 기반으로 사용자 인증 정보를 SecurityContext에 설정합니다.
+   *
+   * @param request 클라이언트의 요청 정보
+   * @param response 서버에서 클라이언트로 보낼 응답 정보
+   * @param filterChain 요청을 처리하는 필터 체인
+   * @throws ServletException 서블릿 예외 발생 시 예외 발생
+   * @throws IOException I/O 처리 중 오류 발생 시 예외 발생
+   */
+  @Override
+  protected void doFilterInternal(
+      HttpServletRequest request,
+      HttpServletResponse response,
+      FilterChain filterChain) throws ServletException, IOException {
+
+    String authorizationHeader = request.getHeader(HEADER_AUTHORIZATION);
+    String token = getAccessToken(authorizationHeader);
+
+    if(tokenProvider.validToken(token)){
```

Comment:
```suggestion
    if (tokenProvider.validToken(token)) {
```
coding convention 상으로는 위와 같은 style 을 지켜주는 것이 좋을 것 같아요. 습관화해보시면 좋겠네요~!

Reply:
말씀 주신 부분 Google Java Style Guide 에서 if 문을 포함한 조건문일때 Horizontal whitespace 부분에서 다시 한번 확인했습니다. 

5ee19e85 : chore : coding convention에 맞게 수정